package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+
+class Pca:
+    def __init__(self, universe, atom_selection=None, start=None, stop=None,
+    step=None, reference_coordinates=None, fitting_tolerance = 0.01,
+    max_iterations = 10, captured_variance = None):
+        """
+        Initialises a new Pca object with the data from the trajectory 
+        associated with the supplied universe, subject to the (optional) 
+        atom and frame selections. The selected atoms are least-squares 
+        fitted using an iterative procedure to remove global rotation and 
+        translation before the covariance matrix is calculated. 
+        If reference_coordinates, in the form of a [N, 3] numpy array (where 
+        N is the number of selected atoms) is provided, this will be used to 
+        initialise the least-squares fitting process, otherwise the first 
+        snapshot in the trajectory will be used.
+
+        :Arguments:
+          *universe*
+            :class:`MDAnalysis.Universe`
+
+          *atom_selection* 
+            a valid selection string for 
+            :meth:`MDAnalysis.core.AtomGroup.AtomGroup.select_atoms`
+
+          *start*
+            int, the index (zero-based) of the first frame in the trajectory
+            to include in the PC analysis
+
+          *stop*
+            int, the index (zero-based) of the last frame in the trajectory 
+            to include in the PC analysis
+
+          *step*
+            int, the increment between frames to be included in the analysis
+
+          *fitting_tolerance*
+            float, the iterative least-squares fitting process will terminate
+            when the average structure has an RMSD less than this from the
+            structure generated in the previous cycle (or if *max_iterations*
+            is reached first).
+
+          *max_iterations*
+            int, the maximum number of least-squares fitting cycles
+
+          *captured_variance*
+            int or float, if < 1 then enough eigenvectors will be kept to
+            explain *captured_variance* fraction of the total variance, if
+            >=1 (and integral), this is the number of eigenvectors that will
+            be kept. Providing this argument permits the precalculation of 
+            projections, which can improve performance.
+        """
+
+        # Step 1: Create the "Fasu" object from the selected trajectory data.
+        self._f = Fasu(universe, atom_selection=atom_selection, start=start, stop=stop, step=step)
+        self.natoms = self._f.natoms
+        self.numframes = self._f.numframes
+
+        # Step 2: Calculate the mean structure, later used to remove global
+        #         translation and rotation from the trajectory frames before
+        #         the covariance matrix calculation. This iterative method
+        #         is based on the "Procrustes" approach (Dryden and Mardia,
+        #         "Statistical Shape Analysis", ISBN 978-0-471-95816-1).
+
+        if reference_coordinates is None:
```

Comment:
I see the points you make about the fitting but I still thing it should be possible to disable it. There might be good reason for people that they don't want to do a fit or they may want to fit the simulation with another algorithm. They might have already fitted a trajectory.  It could also be that I want to fit on a subset of the protein only.

There are a lot of good reason why someone might not want to have the fitting applied.  Like calculating the PCA of a randomwalk.